platform/ios/app/MBXAppDelegate.m

```diff
@@ -11,6 +11,12 @@ @implementation MBXAppDelegate
 
 - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions
 {
+    NSArray *paths = NSSearchPathForDirectoriesInDomains(
+                                                         NSApplicationSupportDirectory,
+                                                         NSUserDomainMask, YES);
+    NSLog(@"%@/com.geolives.maplibre.app/.mapbox/cache.db",[paths objectAtIndex:0]);
```

Comment:
This looks like it is leftover from debugging.

Reply:
Yes, sorry, I have just fixed it.